### ISSUE TITLE
Unity 2022.1.17f1 is not compatible.

### ISSUE THREAD

**Issue by ZaryabMustafa** [tylearymf/UniHacker#61]
![image](https://user-images.githubusercontent.com/10380539/192753303-af5e7171-a059-4afa-b58d-8e6f9e7172d0.png)

**Comment by tylearymf** [comment]
@ZaryabMustafa Can you share the unity binaries?

**Comment by ZaryabMustafa** [comment]
> @ZaryabMustafa Can you share the unity binaries?

Yes here you go: https://drive.google.com/file/d/1d6eu8hjbK44p0u3ta1ppd8XQ0vbAGYbc/view?usp=sharing

**Comment by tylearymf** [comment]
I guess you didn't choose unity.exe.

**Comment by ZaryabMustafa** [comment]
> 

I made a rar file of exe 
![image](https://user-images.githubusercontent.com/10380539/192758359-5ff3b086-948d-413e-9292-bc7a4b07fae2.png)

**Comment by tylearymf** [comment]
All right, I'll check

**Comment by tylearymf** [comment]
@ZaryabMustafa Did you not choose Unity.exe, I tried it and it can be cracked.

![image](https://user-images.githubusercontent.com/14980389/192781308-40f799b6-6601-48fb-9fc8-59dd2695d7f6.png)

**Comment by ZaryabMustafa** [comment]
I did before. I tried it again and now it works. Thank you for the prompt response. I'm closing the issue now.
